BLOG_POSTING_GUIDELINES.md

```diff
@@ -0,0 +1,61 @@
+# Blog contributing guidelines
```

Comment:
This guide is very complete. Try to write everything related to user contribution in `CONTRIBUTING.md` and only keep specific notes here. Probably, those notes can have a section in the README.md for easier and instant access.

Reply:
So you want to remove this file and paste its contents to `CONTRIBUTING.md` ? I can't seem to find anything I find relevant to put in the `README.md`

Reply:
Most of the content, is related to how to contribute to this project in general (feature branch and peer review, for example). However, when you explain that our website has a blog section and all posts are markdown files, that should go to the `README.md`.

Also, we should explain about the need to do a fork if you are note a CoderDojo Braga team member on GitHub.